static/api-spec.yaml

```
@@ -0,0 +1,754 @@
+                                                  'status':
+                                                    { 'type': 'string' },
+                                                },
+                                              'additionalProperties':
+                                                { 'type': 'string' },
+                                            },
+                                          'nullable': true,
+                                        },
+                                      'details':
+                                        {
+                                          'type': 'object',
+                                          'example':
+                                            {
+                                              'database': { 'status': 'up' },
+                                              'redis':
+                                                {
+                                                  'status': 'down',
+                                                  'message': 'Could not connect',
+                                                },
+                                            },
+                                          'additionalProperties':
+                                            {
+                                              'type': 'object',
+                                              'properties':
+                                                {
+                                                  'status':
+                                                    { 'type': 'string' },
+                                                },
+                                              'additionalProperties':
+                                                { 'type': 'string' },
+                                            },
+                                        },
+                                    },
+                                },
+                            },
+                        },
+                    },
+                },
+              'tags': ['System'],
+            },
+        },
+      '/dv/{configHash}':
+        {
+          'get':
+            {
+              'operationId': 'DVController_getClusterDefinition',
+              'summary': 'Retrieve a Distributed Validator Cluster proposal',
+              'description': "This endpoint allows a charon client or launchpad interface to retrieve the terms of a proposed DKG. Once all operators listed in the DKG have submitted signed approvals to the terms, this object will be ready for a cluster of operators to use as part of a DKG ceremony. If the objects in the `operators` array are not fully populated, these operators need to use the [PUT request](#/Distributed-Validators/operation/DVController_updateClusterDefinition) to upload their charon client's public key and a signature from their address to indicate their acceptance of the terms.",
```

```suggestion
              'description': "This endpoint allows a charon client or launchpad interface to retrieve the terms of a proposed DKG. Once all operators listed in the DKG have signed the terms of the cluster, this object will be ready for a cluster of operators to use as part of a DKG ceremony. If the objects in the `operators` array are not fully populated, these operators need to use the [PUT request](#/Distributed-Validators/operation/DVController_updateClusterDefinition) to upload their charon client's public key and a signature from their address to indicate their acceptance of the terms.",
```

Also, the PUT Request link does not work